### PR TITLE
Added callingstationssid and nasidentifierin radius_acct sql table

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Added callingstationssid and nasidentifier in the radacct sql table

# Impacts
RADIUS accounting workflow

# Delete branch after merge
YES

# NEWS file entries

## Enhancements
* Added SSID and NAS identifier in the radacct table (accounting)

### CODE REVIEW

File: db/pf-schema-X.Y.Z.sql [hunk 1]
```diff
@@ -668,6 +668,8 @@ CREATE PROCEDURE acct_start (
     IN p_framedprotocol varchar(32),
     IN p_framedipaddress varchar(15),
     IN p_acctstatustype varchar(25),
+    IN p_nasidentifier varchar(64),
+    IN p_calledstationssid varchar(64),
     IN p_tenant_id int
```
Comment (satkunas) [inline db/pf-schema-X.Y.Z.sql]: int(11) unsigned

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 2]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+    IN p_calledstationid varchar(50),
+    IN p_callingstationid varchar(50),
+    IN p_acctterminatecause varchar(32),
+    IN p_servicetype varchar(32),
+    IN p_framedprotocol varchar(32),
+    IN p_framedipaddress varchar(15),
+    IN p_acctstatustype varchar(25),
+    IN p_nasidentifier varchar(64),
+    IN p_calledstationssid varchar(64),
+    IN p_tenant_id int
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: int(11) unsigned

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 3]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+)
+BEGIN
+
+# We make sure there are no left over sessions for which we never received a "stop"
+DECLARE Previous_Session_Time int(12);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 4]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+--
+-- Adding RADIUS Stop Stored Procedure
+--
+
+DROP PROCEDURE IF EXISTS acct_stop;
+DELIMITER /
+CREATE PROCEDURE acct_stop (
+  IN p_timestamp datetime,
+  IN p_framedipaddress varchar(15),
+  IN p_acctsessiontime int(12),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 5]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_acctinputoctets bigint(20),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 6]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_acctoutputoctets bigint(20),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 7]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_connectinfo_stop varchar(50),
+  IN p_calledstationid varchar(50),
+  IN p_callingstationid varchar(50),
+  IN p_servicetype varchar(32),
+  IN p_framedprotocol varchar(32),
+  IN p_acctterminatecause varchar(12),
+  IN p_acctstatustype varchar(25),
+  IN p_nasidentifier varchar(64),
+  IN p_calledstationssid varchar(64),
+  IN p_tenant_id int
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: int(11) unsigned

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 8]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+)
+BEGIN
+  DECLARE Previous_Input_Octets bigint(20);
+  DECLARE Previous_Output_Octets bigint(20);
+  DECLARE Previous_Session_Time int(12);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 9]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_framedprotocol varchar(32),
+  IN p_acctterminatecause varchar(12),
+  IN p_acctstatustype varchar(25),
+  IN p_nasidentifier varchar(64),
+  IN p_calledstationssid varchar(64),
+  IN p_tenant_id int
+)
+BEGIN
+  DECLARE Previous_Input_Octets bigint(20);
+  DECLARE Previous_Output_Octets bigint(20);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 10]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_servicetype varchar(32),
+  IN p_framedprotocol varchar(32),
+  IN p_acctterminatecause varchar(12),
+  IN p_acctstatustype varchar(25),
+  IN p_nasidentifier varchar(64),
+  IN p_calledstationssid varchar(64),
+  IN p_tenant_id int
+)
+BEGIN
+  DECLARE Previous_Input_Octets bigint(20);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 11]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+--
+-- Adding RADIUS Updates Stored Procedure
+--
+
+DROP PROCEDURE IF EXISTS acct_update;
+DELIMITER /
+CREATE PROCEDURE acct_update(
+  IN p_timestamp datetime,
+  IN p_framedipaddress varchar(15),
+  IN p_acctsessiontime int(12),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 12]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_acctinputoctets bigint(20),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 13]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_acctoutputoctets bigint(20),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 14]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  IN p_acctauthentic varchar(32),
+  IN p_connectinfo_start varchar(50),
+  IN p_calledstationid varchar(50),
+  IN p_callingstationid varchar(50),
+  IN p_servicetype varchar(32),
+  IN p_framedprotocol varchar(32),
+  IN p_acctstatustype varchar(25),
+  IN p_nasidentifier varchar(64),
+  IN p_calledstationssid varchar(64),
+  IN p_tenant_id int
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: int(11) unsigned

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 15]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+)
+BEGIN
+  DECLARE Previous_Input_Octets bigint(20);
+  DECLARE Previous_Output_Octets bigint(20);
+  DECLARE Previous_Session_Time int(12);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 16]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  DECLARE Previous_AcctUpdate_Time datetime;
+
+  DECLARE Opened_Sessions int(12);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 17]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  DECLARE Latest_acctstarttime datetime;
+  DECLARE cnt int(12);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 18]
```diff
@@ -102,4 +102,326 @@ CREATE TABLE key_value_storage (
...
+  DECLARE countmac int(12);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigend?

**Review by satkunas — CHANGES_REQUESTED** [review]

**Comment by fdurand** [comment]
Done

**Review by satkunas — APPROVED** [review]

**Pull request merged**